doc/nrf/known_issues.rst

```diff
@@ -21,7 +21,8 @@ Use the drop-down filter to see known issues for previous releases and check if
 
    <select name="versions" id="versions-select">
      <option value="all">All versions</option>
-     <option value="v1-4-1" selected>v1.4.1</option>
+     <option value="v1-4-2" selected>v1.4.2</option>
```

Comment:
@nordic-hani , kindly please give a look at known issue status for lwm2m_carrier. I've propagated 1.4.2 to all 1.4.1 issues, but some might've been fixed in 0.10.2.